src/common/node/fileSystem.ts

```diff
@@ -1,28 +1,33 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for details.
 
-import * as fs from "fs";
+import * as nodeFs from "fs";
 import * as path from "path";
 import * as Q from "q";
 
 export class FileSystem {
+    private fs: typeof nodeFs;
```

Comment:
I think it's ok as is. For instance variables it makes sense to me to rename `child_process` to `childProcess`, so the import can stay as `child_process` without conflicts. In case of a conflict like for `fs`, I prefer doing the rename at the import variable, since that is only used once in the entire file. I think that makes the rest of the file more readable (`this.fs` as opposed to `this.[prefix]Fs`).